src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/fuzzer.py

```diff
@@ -45,7 +45,8 @@ def get_arguments(fuzzer_path) -> options.FuzzerArguments:
   if timeout is None or timeout > constants.DEFAULT_TIMEOUT_LIMIT:
     arguments[constants.TIMEOUT_FLAGNAME] = constants.DEFAULT_TIMEOUT_LIMIT
 
-  if not rss_limit_mb and utils.is_chromium():
+  if not rss_limit_mb and (utils.is_chromium() or
+                           utils.default_project_name() == 'google'):
```

Comment:
Maybe let's just not special case this? It's going to happen to oss-fuzz too when clang is rolled.

Reply:
Let's be cautious for now imo, in case this has unintended consequences for oss-fuzz.